project/build.properties

```diff
@@ -1,2 +1,2 @@
 # suppress inspection "UnusedProperty"
-sbt.version = 1.4.1
+sbt.version = 1.4.7
```

Comment:
1.5.0?

Reply:
Can't go beyond 1.4.7 since travis CI's sbt launcher is still hardcoded to bintray (which is gone now)